kytos/core/config.py

```diff
@@ -56,6 +56,10 @@ def __init__(self):
                             action='store_true',
                             help="Run in foreground (ctrl+c to stop)")
 
+        parser.add_argument('-ft', '--no_terminal',
```

Comment:
`-ft` and `--no_terminal` is not a UNIX Posix standard. 

Not sure if I understood how this would run in the foreground without a terminal. Also if it is without a terminal how CTRL+C will stop?

Why not using nohup here?

Reply:
Suggestions applied.